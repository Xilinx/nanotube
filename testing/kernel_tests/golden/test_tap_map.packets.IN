###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
# Read key 0, read key 1
2020-06-23 01:00:00.000000
0000 00 41 42 43 00 00 00 00 00 55 55 55 55 55 55 55 55
0011 00 44 45 46 00 00 00 00 00 aa aa aa aa aa aa aa aa
# Insert key 0, read key 0
2020-06-23 02:00:00.000000
0000 01 41 42 43 00 00 00 00 00 61 62 63 64 65 66 67 68
0011 00 41 42 43 00 00 00 00 00 55 55 55 55 55 55 55 55
# Insert key 1, read key 1
2020-06-23 03:00:00.000000
0000 01 44 45 46 00 00 00 00 00 91 92 93 94 95 96 97 98
0011 00 44 45 46 00 00 00 00 00 55 55 55 55 55 55 55 55
# Read key 0, read key 1
2020-06-23 04:00:00.000000
0000 00 41 42 43 00 00 00 00 00 55 55 55 55 55 55 55 55
0011 00 44 45 46 00 00 00 00 00 aa aa aa aa aa aa aa aa
