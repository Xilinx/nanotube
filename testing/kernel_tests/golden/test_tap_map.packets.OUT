###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
# Read key 0 -> absent, read key 1 -> absent
2020-06-23 01:00:00.000000
0000 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
0011 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
# Insert key 0 -> inserted, read key 0 -> present
2020-06-23 02:00:00.000000
0000 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
0011 03 00 00 00 00 00 00 00 00 61 62 63 64 65 66 67 68
# Insert key 1 -> inserted, read key 1 -> present
2020-06-23 03:00:00.000000
0000 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
0011 03 00 00 00 00 00 00 00 00 91 92 93 94 95 96 97 98
# Read key 0 -> present, read key 1 -> present
2020-06-23 04:00:00.000000
0000 03 00 00 00 00 00 00 00 00 61 62 63 64 65 66 67 68
0011 03 00 00 00 00 00 00 00 00 91 92 93 94 95 96 97 98
